proxy.conf.template

```diff
@@ -5,7 +5,9 @@
 # The docker cli uses the realm domain given in the www-authenticate to auth.
 # So we need to re-write from ghcr.io to our proxy.
 map $upstream_http_www_authenticate $auth_header {
-        ~(.*)ghcr.io(.*)   $1${PROXY_DOMAIN}$2;
+        # used to have 1 occurance of domain, now has 2. Handle both.
+        ~(.*)ghcr.io(.*)ghcr.io(.*) $1${PROXY_DOMAIN}$2${PROXY_DOMAIN}$3;
+        ~(.*)ghcr.io                $1${PROXY_DOMAIN}$2;
```

Comment:
uninformed question - should this be `~(.*)ghcr.io(.*)` rather than ` ~(.*)ghcr.io`? if not, you can remove `$2`

Reply:
It absolutely should, good catch, thanks